component/main.jsonnet

```diff
@@ -7,4 +9,9 @@ local params = inv.parameters.vcluster;
 
 // Define outputs below
 {
+  '00_namespace': kube.Namespace(params.namespace),
+} +
+{
+  ['10_cluster_' + name]: cluster.Cluster(name, params.base + com.makeMergeable(params.clusters[name]))
```

Comment:
Why do we reimplement instances in Jsonnet instead of using component instances?

Reply:
Because i found it painful to use if you need many clusters with a common baseline setup.

Reply:
Multi instance components have support for a common base configuration.

Reply:
Then I might have missed something 😅 thanks for mentioning

Reply:
basically, the base config for a multi instance component is in key `<component-name>` and overrides are in key `<instance-name>`, see https://syn.tools/commodore/reference/architecture.html#_component_instantiation